api/build.gradle

```diff
@@ -13,3 +13,10 @@ javadoc {
     exclude 'io/opencensus/internal/**'
     exclude 'io/opencensus/trace/internal/**'
 }
+
+animalsniffer {
```

Comment:
Not sure that this will include the auto-value classes.

Reply:
Copied this from gRPC-Java: https://github.com/grpc/grpc-java/blob/master/core/build.gradle#L49-L55.  /cc @zhangkun83 any suggestion on how to enforce `animalsniffer` on generated classes?

Reply:
/cc @ericgribkoff too - looks like `animalsniffer` to Android 14 is added to gRPC in https://github.com/grpc/grpc-java/pull/4727. 

Any help is high appreciated!